### PR TITLE
Update turn-audit-log-search-on-or-off.md

### PR DESCRIPTION
**Pull request by sujitnaray**

### CODE REVIEW

**Comment by sujitnaray** [comment]
@markjjo

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/turn-audit-log-search-on-or-off.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ description: How to turn on or off the Audit log search feature in the Security
 
 # Turn audit log search on or off
 
-You (or another admin) must turn on audit logging before you can start searching the audit log. When audit log search in the Security & Compliance Center is turned on, user and admin activity from your organization is recorded in the audit log and retained for 90 days, and up to one year depending on the license assigned to users. However, your organization may have reasons for not wanting to record and retain audit log data. In those cases, a global admin may decide to turn off auditing in Microsoft 365.
+Audit Logging is turned on for enterprise customers, by default. When audit log search in the Security & Compliance Center is turned on, user and admin activity from your organization is recorded in the audit log and retained for 90 days, and up to one year depending on the license assigned to users. However, your organization may have reasons for not wanting to record and retain audit log data. In those cases, a global admin may decide to turn off auditing in Microsoft 365.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/turn-audit-log-search-on-or-off.md]:
```suggestion
Audit logging is turned on for Enterprise customers, by default. When audit log search in the Security & Compliance Center is turned on, user and admin activity from your organization is recorded in the audit log and retained for 90 days, and up to one year depending on the license assigned to users. However, your organization may have reasons for not wanting to record and retain audit log data. In those cases, a global admin may decide to turn off auditing in Microsoft 365.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Comment by sujitnaray** [comment]
@JohanFreelancer9  - Thanks, accepted!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by markjjo** [comment]
@sujitnaray PR merged. Updated article should be live by end of the day.

**Pull request merged**